distributions/demo-google-deployment/bin/setup_gke_environment.sh

```diff
@@ -345,7 +345,8 @@ gcloud compute url-maps create ${LB_NAME} \
 --default-service ${API_BACKEND_SERVICE_NAME}
 gcloud compute url-maps add-path-matcher ${LB_NAME} \
 --default-service ${API_BACKEND_SERVICE_NAME} --path-matcher-name "static-bucket-mapping" \
---backend-bucket-path-rules "/static/*=${STATIC_BUCKET_NAME}"
+--backend-bucket-path-rules "/static/*=${STATIC_BUCKET_NAME},/index.html=${STATIC_BUCKET_NAME}" \
+--backend-service-path-rules "/api/*=${API_BACKEND_SERVICE_NAME}"
```

Comment:
Why is api here?

Reply:
Updated. Now that we use a "website bucket" for the static bucket, we can set the default page there to index.html and default the LB to the bucket. then we need /api/* to map to the API backend.